docs/data-types/mysql.md

```diff
@@ -139,8 +139,8 @@ and `ON UPDATE` actions, we drop them.
 
 ## Default Values
 
-While Spanner supports default values, Spanner migration tool currently does not support translating source `DEFAULT` constraints to Spanner `DEFAULT` constraints. We drop the `DEFAULT` MySQL constraint during conversion.
-It can be manually added to the DDL via an `ALTER TABLE` command.
+Spanner Migration Tool will migrate all valid `DEFAULT` values from MySQL source automatically
+to Spanner and drop invalid `DEFAULT` constraints.
```

Comment:
we should mention that a warning will be displayed for dropped/invalid DEFAULT constraints. Also I think calling existing default constraints invalid is wrong, we should say that we map 1:1 and in case the syntax is invalid for spanner we drop the constraint